parquet-thrift/src/main/java/parquet/hadoop/thrift/ThriftReadSupport.java

```diff
@@ -73,6 +73,12 @@ public static void setRecordConverterClass(JobConf conf,
     conf.set(RECORD_CONVERTER_CLASS_KEY, klass.getName());
   }
 
+  public static void setRecordConverterClass(Configuration conf,
```

Comment:
yeah, Ideally we'd remove the JobConf one because it is less general than the Configuration one.
I'll mark it as deprecated. (a JobConf is a Configuration, but a Configuration is not a JobConf).